s3-eventbridge-glue/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+    "title": "Automatic Data Transformation and Catalog with Amazon S3 notifications and AWS Step functions",
+    "description": "This pattern facilitates the cataloging and transformation of CSV data, using an event-driven architecture to automatically trigger data cleaning and transformation jobs",
+    "language": "Python",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "When you upload a .csv file to the Input S3 bucket, it matches with an Amazon EventBridge rule that triggers a State Machine. The State Machine is composed of two steps:",
+        "1. Starts a AWS Glue Crawler to crawl the input bucket. This allows you to have the raw data (before transformation) inside the Data Catalog",
+        "2. Start a AWS Glue ETL Job, that runs a simple transformation job and drops empty columns in the dataset" ,
+        "The processed/transformed data is then added to the Data Catalog and to an S3 bucket to allow for further processing."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-eventbridge-glue",
+        "templateURL": "serverless-patterns/s3-eventbridge-glue",
```

Comment:
`gitHub.template.templateURL`: gitHub.template.templateURL Please remove the projectFolder value from the templateFile

Reply:
Not sure what you mean by that?

Reply:
Resolved